website/components/Seo.tsx

```diff
@@ -29,6 +29,11 @@ export interface Props {
   themeColor?: string;
   /** @title Canonical URL */
   canonical?: string;
+  /**
```

Comment:
I think we don't need this. It's ok always to consider all pages on indexing, i.e. this should always be true